node/child_process.ts

```diff
@@ -140,6 +143,49 @@ export function spawn(
   return new ChildProcess(command, args, options);
 }
 
+function validateTimeout(timeout?: number) {
+  if (timeout !== undefined && !(Number.isInteger(timeout) && timeout >= 0)) {
```

Comment:
To match the Node implementation:

```suggestion
  if (timeout != null && !(Number.isInteger(timeout) && timeout >= 0)) {
```